admin/services/marking.service.js

```diff
@@ -51,7 +53,7 @@ markingService.mark = async function (completedCheck) {
   await completedCheckDataService.save(completedCheck)
 
   // update the check meta info
-  await checkDataService.update({checkCode: completedCheck.data.pupil.checkCode}, {'$set': {results: results}})
+  await checkDataService.sqlSetResults(completedCheck.data.pupil.checkCode, results.marks, results.maxMarks, results.processedAt)
```

Comment:
I found this name a little odd.   How about  `sqlUpdateCheckWithResults()` ?

Reply:
👍